utils/rpms/sysctl.d/10-daos_server.conf

```diff
@@ -0,0 +1,2 @@
+net.core.rmem_max=104857 +
+net.core.rmem_default=104857
```

Comment:
This file should just be in `utils/rpms` directory.  No need for a subdir.  I don't think `rpmbuild` will even find if it's in a subdir.